google-cloud-dialogflow-cx/pom.xml

```diff
@@ -97,6 +97,11 @@
       <classifier>testlib</classifier>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>google-cloud-dialogflow-cx-parent</artifactId>
+      <version>${project.version}</version>
+    </dependency>
```

Comment:
what is this new dependency for? I don't see if being used in the file below.

Reply:
Oh that should be not there. I will delete that.